src/ServiceControl.Audit.Persistence.RavenDB/ServerConfiguration.cs

```diff
@@ -1,6 +1,8 @@
 ﻿namespace ServiceControl.Audit.Persistence.RavenDB
 {
-    public class ServerConfiguration
+    using ServiceControl.RavenDB;
+
+    public class ServerConfiguration : IRavenClientCertificateInfo
```

Comment:
Please don't hesitate to ignore. I do not find the interface is particularly helpful. We could also just have a record/class encapsulating these settings that is then passed to the reader code